modules/Bio/EnsEMBL/Compara/DBSQL/GeneMemberAdaptor.pm

```diff
@@ -254,6 +254,16 @@ sub store {
 
 }
 
+sub delete {
+    my ($self, $gene_member) = @_;
+    foreach my $seq_member (@{$gene_member->get_all_SeqMembers}) {
+        $seq_member->adaptor->delete($seq_member);
+    }
+    $self->dbc->do('DELETE FROM gene_member_qc          WHERE gene_member_stable_id = ?', undef, $gene_member->stable_id);
+    $self->dbc->do('DELETE FROM member_xref             WHERE gene_member_id = ?', undef, $gene_member->dbID);
+    $self->dbc->do('DELETE FROM gene_member_hom_stats   WHERE gene_member_id = ?', undef, $gene_member->dbID);
```

Comment:
Shouldn't `ortholog_goc_metric` be also targeted?

Reply:
Yes it should. But also, since @carlacummins rewrote the GOC analyses this table is not populated any more. I would actually suggest to remove it from our production schema